cmd/server/flags.go

```diff
@@ -439,6 +439,52 @@ var flags = append([]cli.Flag{
 		Usage:   "gitlab skip ssl verification",
 	},
 	//
+	// Bitbucket Server (previously Stash)
```

Comment:
Is it only supporting bitbucket server or also datacenter. As server reaches EOL it would be great if it works with the datacenter version as well and I would prefer to name it `bitbucket-datacenter` then as it seems Atlassian seems to plan to sell cloud or datacenter in the future (we could still try to support bitbucket-server as long as possible with the datacenter version as well if both share the same api).

Reply:
The API on datacenter should be the same but I am pretty sure we are running the "server" edition currently not the "datacenter" - the API documentation is the same though.

Reply:
I'll check with our Atlassian team - we might actually be running the datacenter version already - the "about" page in our Bitbucket does not seem to reveal this :-)